rfc6962-bis.xml

```diff
@@ -84,9 +84,11 @@ do not need to be trusted because they are publicly auditable. Anyone may verify
 the correctness of each log and monitor when new certificates are added to it.
 The logs do not themselves prevent misissue, but
 they ensure that interested parties (particularly those named in certificates)
-can detect such misissuance. Note that this is a general mechanism; but in this
-document, we only describe its use for public TLS server certificates issued by
-public certification authorities (CAs).
+can detect such misissuance. Note that this is a general mechanism that could
+be used for transparently logging any form of binary data, subject to some kind
+of inclusion criteria. In this document, we only describe its use for public
+TLS server certificates, i.e. where the inclusion criteria is a valid
```

Comment:
the "i.e." clause should be in parentheses.

Also, apparently "All but one main style guide recommends a comma after use: i.e., and e.g.,"
http://www.instructionalsolutions.com/blog/bid/77073/How-to-Use-i-e-and-e-g-Correctly-in-Business-Writing